photon-core/src/main/java/org/photonvision/vision/pipe/impl/ArucoDetectionPipe.java

```diff
@@ -43,8 +44,13 @@ public class ArucoDetectionPipe
 
     @Override
     protected List<ArucoDetectionResult> process(CVMat in) {
+        if (in.getMat().empty()) return List.of();
+
         var imgMat = in.getMat();
-        var detections = photonDetector.detect(imgMat);
+
+        // var detections = photonDetector.detect(imgMat);
+        var detections = ArucoNanoDetector.detect(in);
```

Comment:
Needs a toggle between the two

Reply:
Moto moto was working on this. They also claim this is broken on windows